conf/trigger_settings.py

```diff
@@ -279,6 +279,12 @@
 # Assign NETDEVICES_SOURCE to NETDEVICES_FILE for backwards compatibility
 NETDEVICES_FILE = NETDEVICES_SOURCE
 
+# TextFSM Vendor Mappings. Override this if you have defined your own TextFSM templates.
```

Comment:
Please add this to `trigger/conf/global_settings.py` please so that it is a default global setting that doesn't need to be overidden.